lib/Doctrine/Migrations/Metadata/AvailableMigration.php

```diff
@@ -8,7 +8,7 @@
 use Doctrine\Migrations\Version\Version;
 
 /**
- * The class AvailableMigration an available migration.
+ *  Available migrations may or may not be already executed
```

Comment:
Looks like I messed up the spacing
```suggestion
 * Available migrations may or may not be already executed
```